docs/rst/tutorials/core/qos/xml_profiles/xml_profiles.rst

```diff
@@ -0,0 +1,400 @@
+.. include:: ../../../../exports/alias.include
+
+.. _tutorials_xml_profiles_intro:
+
+Configuring Fast-DDS QoS via XML profiles
+=========================================
+
+.. contents::
+    :depth: 2
+    :local:
+    :backlinks: none
+
+XML configuration
+-----------------
+
+Fast DDS over Vulcanexus offers the possibility of fully configuring QoS policy through XML profile definition.
+This can be done in several ways depending on whether the application used is a native Fast DDS application or a ROS 2 application
+using Fast DDS's RMW implementation.
+This guide will focus on the latter.
+For more information regarding XML profiles in Fast DDS please refer to the
+`XML configuration page <https://fast-dds.docs.eprosima.com/en/latest/fastdds/xml_configuration/xml_configuration.html>`_
+from Fast DDS's documentation.
+
+.. _vulcanexus_configure:
+
+Configuring *Fast DDS* in Vulcanexus for ROS 2 applications
```

Comment:
I guess
```suggestion
Configuring *Fast DDS* in Vulcanexus for Vulcanexus applications
```

Reply:
What about `Configuring *Fast DDS* in Vulcanexus applications`?

Reply:
I think it pretty much has the same issue. In my opinion, if someone is migrating from ROS2 to Vulcanexus he is not going to search for Vulcanexus applications, he is going to search for ROS2 since that's what he already has. Additionally, someone searching for ROS2 applications may find out about Vulcanexus that way.